tests/unit/array/storage/elastic/test_del.py

```diff
@@ -1,17 +1,25 @@
-from docarray import Document, DocumentArray
 import pytest
+import string
+import random
+
+from docarray import Document, DocumentArray
 
 
 @pytest.mark.filterwarnings('ignore::UserWarning')
 @pytest.mark.parametrize('deleted_elmnts', [[0, 1], ['r0', 'r1']])
-def test_delete_offset_success_sync_es_offset_index(deleted_elmnts, start_storage):
+@pytest.mark.parametrize('columns', [[('price', 'int')], {'price': 'int'}])
+def test_delete_offset_success_sync_es_offset_index(
+    deleted_elmnts, start_storage, columns
+):
+    letters = string.ascii_lowercase
+    random_name = ''.join(random.choice(letters) for _ in range(5))
     elastic_doc = DocumentArray(
         storage='elasticsearch',
         config={
             'n_dim': 3,
-            'columns': [('price', 'int')],
+            'columns': columns,
             'distance': 'l2_norm',
-            'index_name': 'test_delete_offset_success_sync_es_offset_index',
+            'index_name': random_name,
```

Comment:
```suggestion
```
let's just remove the name it will create a random name (to be applied elsewhere)